cpp/ps0002/main (3).cpp

```diff
@@ -2,15 +2,17 @@
 using namespace std;
 int main()
 {
-    int first=27;
-    int second=2;
-    int Team =10;
-     double better = 20.5;
-     char ch = 'A';
-
-    double age = 333.5;
-    short number = 1;
-    double Line = 444.4;
-    double Age =444.4 ;
-     return 0;
-}
\ No newline at end of file
+    short population = 27; // 0.25/0.75
+    short drug_alternatives = 2; // 0.75/0.75 (repeated mistakes)
+    short Team = 10; // 0.75/0.75 (repeated mistake)
+    float pie = 20.5; // 0.5/0.75 (+repeated mistake)
+    char ch = 'A'; // 0.75/0.75
+    int births = 1626556; // 0.25/0.75 (missed)
+    double number = 34.56467788; // 0.25/0.75
+    float Line = 444.4; // 0.5/0.75
+    // double Age = 444.4; (repeated)
+    unsigned int age = 33353436; // 0.25/0.75
+    // total 4.25/6.75
```

Comment:
- [x] all variables required in the document are declared
- [ ] variables are declared with the correct data type
- [x] using correct literals and notations for assigning initial values to the variables